pkg/build/node/cni.go

```diff
@@ -97,6 +92,12 @@ rules:
     verbs:
       - list
       - watch
+  - apiGroups:
+      - ""
+    resources:
+      - pods
```

Comment:
??

Reply:
unrelated , is a leftover that I forget to clean because when I tried to test I used an image with the bug that doesn´t show the pod_IP as an environment variable and I was thinking that was related to the PSP.